test/fuzz/transformation_composite_extract_test.cpp

```diff
@@ -105,9 +105,10 @@ TEST(TransformationCompositeExtractTest, BasicTest) {
                    .IsApplicable(context.get(), transformation_context));
 
   // Id for composite is not a composite.
-  ASSERT_FALSE(TransformationCompositeExtract(
-                   MakeInstructionDescriptor(36, SpvOpIAdd, 0), 200, 27, {})
-                   .IsApplicable(context.get(), transformation_context));
+  ASSERT_FALSE(
+      TransformationCompositeExtract(
+          MakeInstructionDescriptor(37, SpvOpAccessChain, 0), 200, 32, {})
+          .IsApplicable(context.get(), transformation_context));
```

Comment:
I assume this assertion was testing the wrong thing?

Reply:
Yes. Two problems with this one:
1. `MakeInstructionDescriptor(36, SpvOpIAdd, 0)` doesn't exist - there is `%36` but it's `OpAccessChain`.
2. `%27` doesn't dominate `%36`.

Thus, this test failed but not because `%27` is a scalar.